ember-toucan-core/src/components/form/file-input-field.hbs

```diff
@@ -0,0 +1,110 @@
+<div
+  class="flex flex-col {{if @isDisabled 'text-disabled'}}"
+  data-root-field={{if @rootTestSelector @rootTestSelector}}
+>
+  <Form::Field as |field|>
+    <field.Label for={{field.id}} data-label>
+      {{@label}}
+      {{#if @hint}}
+        <field.Hint data-hint>{{@hint}}</field.Hint>
+      {{/if}}
+
+      <field.Control class="flex">
+        <div
+          class="bg-overlay-1 shadow-focusable-outline focus-within:shadow-focus-outline relative inline-flex h-8 w-full flex-wrap items-center rounded-sm"
```

Comment:
I'm wondering if we can move some of these classes from the `<div>` to `<field.Control` to prevent another wrapping div element?

Also wondering if some of these classes are needed at all?  For example, do we need `w-full` and `flex-wrap` here?  Our other inputs/textareas do _not_ set `w-full`, so you may want some padding.

```html
class="bg-overlay-1 shadow-focusable-outline focus-within:shadow-focus-outline relative inline-flex h-8 items-center rounded-sm"
```

Reply:
Changed styles with @ynotdraw